### ISSUE TITLE
What is the scope? / Not usable to capture and organise tasks

### ISSUE THREAD

**Issue by cruftex** [vishal2376/snaptick#83]
I love the idea to combine task management and pomodoro timer. So I installed and gave it a try.

When trying to add a new task I get this screen:

<img src="https://github.com/user-attachments/assets/3cfbaea6-6fb2-48d3-b0ff-31fc46745ee0" width="300">

Unfortunately, this does not help me in any way and immediately put me off. The additional information I can enter here, is not what I need 99% of the time when I want to capture a task.

Of course, different people have different needs and different workflows. Much thought and work has gotten into this app. It should be more clear what the scope of the app actually is. Please put relevant screenshots in the readme and describe what use cases you actually want to solve.

At the moment, this looks more like a calendar app with a different UI and pomodoro timer not a "generic" task manager.

Before tasks can be planned, they need to be captured organised and sorted. David Allen in Getting Things Done speaks about the phases Capture, Clarify, Organise, Reflect, Engage.

What I need for my daily task management, for example:

I want to capture that I need to buy milk at the supermarket. So I would like to add a task "buy milk" and tag it with "@grocer". Next time I stop at a grocer, I list everything tagged with "@grocer" sort it so all veggies and dairy products are clustered together and then, when I put an item  in the cart, I tick off each item from the task list. The app could also tell me what are the most common items I had assigned with "@grocer" in the past, maybe I forgot something.

I like to capture  that I need to book a flight. So I would like to add a task "book flight" with a due date of one week before I fly.

On one hand, maybe the scope of the app is different from what I want. However, on the other hand, before we can plan the activities for a day or a week, where Snaptick might be useful,  we need to capture and sort things somewhere also.

**Comment by vishal2376** [comment]
Hey @cruftex,  

First of all, ❤️  thank you for checking out the app! I really appreciate your honest feedback.  

I’d like to address some of the confusion: 
If you carefully read the description: 

> *Snaptick is a free `daily` task planner app designed to help users increase productivity. With an easy-to-use interface and features like a Pomodoro timer, users can easily organize and achieve their `daily plans`.*

The main goal of this app is to encourage users to wake up in the morning and **plan their entire day within the app**. It’s specifically designed for people who want to stay focused and manage their time effectively—whether it’s studying, completing assignments, or organizing their daily activities. 

This app is not meant to be a generic to-do list or reminders app; it’s tailored for those who want to plan and execute their day with precision and time management. 

Thanks again ❤️🙂
